src/browser/tsconfig.json

```
@@ -18,7 +18,7 @@
   "include": [
     "./**/*",
     "../../typings/xterm.d.ts"
-  ],
+],
```

I might have caused this 🤷 

```suggestion
  ],
```